### PR TITLE
feat: Signup and Login page DONE

### PR DESCRIPTION
**Pull request by maitrakhatri**
# Things added 
- Sign Up Page
- Log In Page

# Screenshots
Screenshot 2022-02-10 at 11-36-17 gozo Store Sign-Up
Screenshot 2022-02-10 at 11-36-23 gozo Store Log-In

### CODE REVIEW

File: Screens/Authentication/log-in/log-in.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                        <label for="email">Email address</label>
+                        <input type="email" name="email" id="" placeholder="admin@gozo.com" required/>
+                    </div>
+                    <div class="input-group">
+                        <label for="password">Password</label>
+                        <input type="password" name="password" id="" required/>
+                    </div>
+                    <div class="login-options">
+                        <input type="checkbox" name="TnC" id="">Remember me <br>
+                        <a href="#">Forgot your Password?</a>
```
Comment (Aditi0505) [inline Screens/Authentication/log-in/log-in.html]: Suggestion: Please provide an id or remove if not needed

**Review by Aditi0505 — APPROVED** [review]
Looks good to me. Just added one comment as suggestion.

File: Screens/Authentication/log-in/log-in.html (hunk 1 above)
Comment (maitrakhatri) [reply Screens/Authentication/log-in/log-in.html]: > Suggestion: Please provide an id or remove if not needed

okay, I'll take care of it

**Review by anupkgurung — APPROVED** [review]
looks good.

**Pull request merged**